RetrospectiveExtension.Frontend/components/feedbackItem.tsx

```diff
@@ -934,7 +936,7 @@ export default class FeedbackItem extends React.Component<IFeedbackItemProps, IF
               }
               {this.state.searchedFeedbackItems
                 .map((columnItem) => {
-                  const feedbackItemProps = FeedbackColumn.createFeedbackItemProps(
+                  const feedbackItemProps = FeedbackColumnStatic.createFeedbackItemProps(
```

Comment:
same with the previous comment

Reply:
done, created `FeedbackColumnHelper` class